README.md

```diff
@@ -1,101 +1,54 @@
 # Contiv Installation for Docker Swarm & Kubernetes 1.4+
-
-## Docker Swarm installation
-
-### Pre-requisites
-* Supported operating systems are CentOS 7.x or RHEL 7.x.
-* Docker needs to be installed on the host where the installer is being run.
-* Install a docker swarm cluster (or)
-  Use the Contiv installer to install the scheduler stack after installing docker
-  on a node external to the cluster where the scheduler stack is being installed.
+Install Contiv on your Docker Swarm or Kubernetes cluster.
+## Docker Swarm Installation
+### Prerequisites
+* CentOS 7.x operating system
+* Docker installed on the host where you are running the installer.
+* Install a Docker Swarm cluster or use the Contiv installer to install the scheduler stack after installing Docker on a node external to the cluster where the scheduler stack is being installed.
 
 ### Contiv Installation
```

Comment:
Can we put something like this if we are removing this diagram :

https://github.com/gaurav-dalvi/scripts/blob/master/contiv-related/Installer.png